packages/core/examples/buttonsExample.tsx

```diff
@@ -48,12 +51,24 @@ export class ButtonsExample extends BaseExample<IButtonsExampleState> {
             <div className="docs-react-example-column">
                 <code>Button</code><br/><br/>
                 <Button
-                    {...removeNonHTMLProps(this.state, INVALID_HTML_PROPS)}
-                    className={classNames(classes, { "docs-wiggle": this.state.wiggling })}
-                    iconName="refresh"
-                    intent={this.state.intent}
-                    onClick={this.beginWiggling}
-                    text="Click to wiggle"
+                  {...removeNonHTMLProps(this.state, INVALID_HTML_PROPS)}
```

Comment:
use 4-space indentation please, here and below (I suggest configuring EditorConfig)